core/components/atoms/dropdown/DropdownList.tsx

```diff
@@ -230,6 +230,11 @@ const DropdownList = (props: OptionsProps) => {
 
   const [popoverStyle, setPopoverStyle] = React.useState<PopoverProps['customStyle']>();
   const [cursor, setCursor] = React.useState(firstEnabledOption);
+  const [minHeight, setMinHeight] = React.useState<number | undefined>();
+  const [popoverOpen, setPopoverOpen] = React.useState(false);
```

Comment:
Can you please elaborate on why we need another state `popoverOpen` when we already have `dropdownOpen`?

Reply:
DropdownOpen upsate always when popover opens because of that offsetHeight changes everytime.